### PR TITLE
Adding clarification to user type conversion

### PR DESCRIPTION
**Pull request by somakbhattacharyya**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/edit-guests-information.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ ms.custom:
 
 You can edit guest information from the Microsoft 365 admin center or the Azure Active Directory portal.
 
+> [!NOTE]
+> In tenant user type conversation of existing guest users is currently not supported in Teams.
```
Comment (JohanFreelancer9) [inline Teams/edit-guests-information.md:30]:
```suggestion
> Editing of in-tenant user-type conversations of existing guest users is currently not supported in Teams.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @somakbhattacharyya, kindly commit the suggested change ONLY IF my interpretation is correct. Your original note is not really clear to me. Many thanks.

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@scanum this PR has unresolved edits. @somakbhattacharyya can you clarify what the note says? I'm not sure what in-tenant user type conversations are.

**Pull request closed without merging**